src/art/ingridclaire/style.css

```diff
@@ -0,0 +1,11 @@
+* {
+  margin: 0;
+  padding: 0;
+}
+
+
```

Comment:
Please reduce it to single line spacing. You can leave line 4 blank and clean other empty lines (that is lines 5 ,6, and 7). It makes your code more neater. Thank you

Reply:
removed!